hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestQuotaThrottle.java

```diff
@@ -652,4 +655,39 @@ public void testExceedThrottleQuota() throws Exception {
     admin.setQuota(QuotaSettingsFactory.unthrottleTable(TABLE_NAMES[0]));
     triggerTableCacheRefresh(TEST_UTIL, true, TABLE_NAMES[0]);
   }
+
+  @Test
+  public void testSetAndGetAllThrottleTypes() throws Exception {
+    for (ThrottleType throttleType : ThrottleType.values()) {
+      canSetAndGetUserThrottle(throttleType);
+    }
```

Comment:
By validating this for every throttle type, we'll prevent new throttle types from missing this key usability piece down the road